database/table.go

```diff
@@ -554,3 +551,17 @@ func (t *Table) Indexes() (map[string]Index, error) {
 
 	return indexes, nil
 }
+
+// PrintIndexes prints a list of indexes of the table receiver.
+func (t *Table) PrintIndexes() error {
```

Comment:
Printing the indexes is a feature that is really linked to the shell. Thus, I suggest to move this logic to its package. 
I think that it's useful to have this function only there. What do you think?

Reply:
I got it.